packages/flutter_driver/lib/src/driver/timeline_summary.dart

```diff
@@ -196,7 +192,7 @@ class TimelineSummary {
 
   List<TimedEvent> _extractGpuRasterizerDrawEvents() => _extractBeginEndEvents('GPURasterizer::Draw');
 
-  List<Duration> _extractFrameDurations() => _extractDurations('Frame');
```

Comment:
Nit: It seems that we can preserve the signature of `_extractFrameDurations` and just change its implementation. Is there any specific reason that we must have a separate `_extractFrameEvents` function?

Reply:
changing the implementation is good suggestion. _extractFrameEvents  just keep same style of _extractGpuRasterizerDrawEvents. I have updated PR, remove redundant code and reduced method nested calls